src/components/FieldGroup.js

```diff
@@ -112,7 +113,21 @@ export const FieldGroup = props => {
     theme,
     dataId,
   } = props;
-  const selectedField = getSelectedField(type, values, selectedValues, 'value');
+  const uniqueValues =
+    values.length > 0
+      ? [
+          ...values.map(value => ({
+            ...value,
+            uuid: uniqueId(value.id),
```

Comment:
Looks appropiated to change uuid as loadash uniqueId does not provide an uuid

Reply:
Makes sense 👍